editor/services/directives.ts

```diff
@@ -30,7 +30,7 @@ class Tooltip {
     text: string;
     opacity: number;
 
-    constructor(private element: ElementRef) {}
+    // constructor(private element: ElementRef) {}
```

Comment:
Is there any reason which justify this line has not been deleted rather than commented?